### ISSUE TITLE
Would it be possible to move the virtual repeat container from the bottom of the document to the bottom of the directive in autocomplete

### ISSUE THREAD

**Issue by jrgleason** [angular/material#6428]
I am currently working to test my autocomplete code, a problem arises when the options under `md-virtual-repeat-container` are at the bottom of the document as opposed to inside the `md-autocomplete`. This is a problem when you have your tests set up with sections in mind, which allows you to limit to just the tags within say md-autocomplete.

I tried looking through the code to see where this happens but I can't figure out where it splits things

**Comment by kseamon** [comment]
This is really more of an autocomplete question than a virtual repeat one. That said, I'm guessing that it is attached to the document so that it does not get clipped by overflow: hidden elements that may surround the autocomplete's input.

I agree that this is a pain from a debugging perspective (Trust that I've spent a lot of time debugging that thing over the last several months). The easiest way I've found to deal with it is to drop a breakpoint into the directive code after it is shown.
